src/components/shared/chart/Canvas.js

```diff
@@ -269,11 +277,20 @@ export class ChartCanvas<HoveredItem> extends React.Component<
 
     const maybeHoveredItem = this.props.hitTest(this._offsetX, this._offsetY);
     if (maybeHoveredItem !== null) {
-      this.setState({
-        hoveredItem: maybeHoveredItem,
-        pageX: event.pageX,
-        pageY: event.pageY,
-      });
+      if (this.state.selectedItem === null) {
+        // Update both the hovered item and the position of the tooltip.
+        this.setState({
+          hoveredItem: maybeHoveredItem,
+          pageX: event.pageX,
+          pageY: event.pageY,
+        });
+      } else {
+        // If there is a selected item, only update the hoveredItem and not the
+        // pageX and pageY values which is used for the position of the tooltip.
+        this.setState({
+          hoveredItem: maybeHoveredItem,
+        });
+      }
```

Comment:
initially I was thinking: we're in a mousemove event handler that can fire a lot, and this state change depends on anothoer state, so maybe we should use a parametric setState call (`setState(state => { new state })`). But the new state doesn't depend on the changed properties, only from other properties, so I think its OK.